docs/agent-policy.md

```diff
@@ -0,0 +1,169 @@
+# Elastic Agent Policies
+
+The policy of an Agent is the user-controlled portion of the Agent configuration that, together with [component spec files](component-specs.md), determines the components that Agent runs and their configuration.
+
+A __component__ is an executable that Agent runs or monitors. That component is divided into __units__ corresponding to its functional behavior. For example, Filebeat is a component, but Filebeat may be run with multiple `filestream` inputs, and each of those is a unit with its own configuration.
+
+Components run by Agent have one or more input units describing the data they collect, and one output unit describing where the data goes.
+
+## Example: policies mapped to running processes
+
+Here is an example policy, and a diagram of the resulting components. (This is a simplified example, for details on the meaning of various fields see the next section.)
+
+```yml
+version: 2
+outputs:
+  elasticsearch1.type: "elasticsearch"
+  elasticsearch2.type: "elasticsearch"
+  logstash.type: "logstash"
+inputs:
+  - type: filestream
+    id: filestream-1
+    use_output: elasticsearch1
+    ...
+  - type: filestream
+    id: filestream-2
+    use_output: elasticsearch1
+    ...
+  - type: metrics
+    id: metrics-1
+    use_output: elasticsearch1
+    ...
+  - type: metrics
+    id: metrics-2
+    use_output: elasticsearch1
+    ...
+  - type: udp
+    id: udp-1
+    use_output: logstash
+    ...
+  - type: endpoint
+    id: endpoint-1
+    use_output: elasticsearch2
+```
+
+![Example deployment without shipper](diagrams/components-example.svg)
+
+In this example, we have defined three outputs: `elasticsearch1`, `elasticsearch2`, and `logstash`. We have also defined 6 total inputs, four writing to `elasticsearch1` and one each writing to `elasticsearch2` and `logstash`.
+
+Agent has divided these inputs among four __components__ (running processes) according to their type. Each component is broken up into multiple __units__, an output unit that sends its data to the appropriate destination, and some number of input units that provide data to the output. For example the `filebeat1` component has three, two input and one output.
+
+
```

Comment:
+1 The various ways to enable the shipper are described in https://github.com/elastic/kibana/issues/141508#issuecomment-1307665963, which we really need to document somewhere permanent because it just lives in the code right now.

Reply:
The policy yaml was identical except for adding `use_shipper: true` but you're right that that was not clear, I have made the config change more explicit